EDDI/MainWindow.xaml.cs

```diff
@@ -565,43 +576,85 @@ private void ChangeLog_Click(object sender, RoutedEventArgs e)
             changeLog.Show();
         }
 
-        public static void uploadLog(IProgress<string> progress)
+        public static void prepareLog(IProgress<string> progress)
         {
-            using (WebClient client = new WebClient())
+            try
             {
-                try
+                string IssueLogDir = Constants.DATA_DIR + @"\logexport\";
+                string IssueLogFile = IssueLogDir + @"eddi_issue.log";
+                string DesktopPath = Environment.GetFolderPath(Environment.SpecialFolder.Desktop) + @"\eddi_issue.zip";
+
+                lock (logLock)
                 {
                     progress.Report("");
-                    client.UploadFile("http://api.eddp.co/log", Constants.DATA_DIR + @"\\eddi.log");
-                    progress.Report("done");
-                    if (false) // temporarily disabled while we fix #91
+
+                    // Create a temporary issue log file, delete any remnants from prior issue reporting
+                    Directory.CreateDirectory(IssueLogDir);
+                    File.Create(IssueLogFile);
+                    File.Delete(DesktopPath);
+
+                    // Use regex to isolate DateTimes from the string
+                    Regex recentLogsRegex = new Regex(@"^(\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2})");
+
+                    var log = File.ReadAllLines(Constants.DATA_DIR + @"\\eddi.log");
+                    double elapsed_time = 0;
+
+                    foreach (string line in log)
                     {
                         try
                         {
-                            File.Delete(Constants.DATA_DIR + @"\\eddi.log");
+                            // Parse log file lines so that we can examine DateTimes
+                            string linedatestring = recentLogsRegex.Match(line).Value;
+                            DateTime linedate = DateTime.Parse(linedatestring);
```

Comment:
This isn't working for me because it is assuming local time. As I am on UTC+1 no lines matched.

Need to use DateTime.TryParseExact() and explicitly pass DateTimeStyles.AssumeUniversal.

Reply:
Trying to revise as suggested but cannot get the method suggested to parse. Do you see what I am doing wrong?
```
                            if(DateTime.TryParseExact(linedatestring, "s", CultureInfo.InvariantCulture, DateTimeStyles.AssumeUniversal, out DateTime linedate))
                            {
                                elapsedTime = (DateTime.UtcNow - linedate).TotalHours;

                                // Fill the issue log with log lines from the most recent hour only
                                if (elapsedTime < 1)
                                {
                                    using (StreamWriter file = new StreamWriter(issueLogFile, true))
                                    {
                                        file.WriteLine(line);
                                    }
                                }
                            }
```

Reply:
Odd though, the original version is working perfectly for me.